Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/PR4715Test.dfy

```diff
@@ -0,0 +1,27 @@
+// RUN: %verify --warn-contradictory-assumptions "%s" > "%t"
```

Comment:
Could you add a [line]comment to indicate what this is testing? Apparently the absence of a warning.